sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs

```diff
@@ -417,10 +366,7 @@ private static void CloseLink(RequestResponseAmqpLink link)
             }
             finally
             {
-                if (_isProcessor)
-                {
                     registration.Dispose();
```

Comment:
It looks like the `registration` user was removed with L335.... do we need the variable anymore?

Reply:
Yes, the variable is still used, but some other params can now be removed.